interaction-manager/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "coherent-gameface-interaction-manager",
-  "version": "2.1.2",
+  "version": "2.1.3",
```

Comment:
This is a feature so the version should be `2.2.0`

Reply:
Yeah, true. Initially it was supposed to be a bug fix so I didn't consider it a feature but when I added the overlap I forgot to change the version. :D